src/utils/DaysOfWeek.ts

```diff
@@ -0,0 +1,10 @@
+
+export const DaysOfWeek: Record<string, string> = {
+    "Lunes": "MONDAY",
+    "Martes": "TUESDAY ",
+    "Miércoles": "WEDNESDAY ",
+    "Jueves": "THRUSDAY ",
+    "Viernes": "FRIDAY ",
+    "Sábado": "SATURDAY ",
+    "Domingo": "SUNDAY "
```

Comment:
Ojo con estos espacios

```suggestion
    "Martes": "TUESDAY",
    "Miércoles": "WEDNESDAY",
    "Jueves": "THRUSDAY",
    "Viernes": "FRIDAY",
    "Sábado": "SATURDAY",
    "Domingo": "SUNDAY"
```